packages/grid/_modules_/grid/components/toolbar/DensitySelector.tsx

```diff
@@ -39,7 +39,7 @@ export function DensitySelector() {
     },
   ];
 
-  const getSelectedDensityIcon = React.useCallback((): React.ReactElement => {
```

Comment:
I needed to update this because the linter was failing, I'm not sure how the previous PR had a green build 🤔

Reply:
Rebase on *master*, it should do it.